kubespray/configurations/all/etcd.yml

```diff
@@ -16,6 +16,9 @@ etcd_data_dir: /var/lib/etcd
 etcd_deployment_type: host
 
 etcd_supported_versions:
+  v1.29: "v3.5.12"
+  v1.28: "v3.5.12"
+  v1.27: "v3.5.12"
```

Comment:
I would set `v3.5.14` for each version, otherwise we will use a older version of etcd compared to what we have in our `v1.25` Kubernetes clusters.

A few months ago, we updated etcd to `v3.5.14` to fix a bug. See this commit: https://github.com/Ferlab-Ste-Justine/terraform-null-kubernetes-installation/commit/4835919f84f51454e668e867fba142dd8bdf766c